### PR TITLE
Add Early Childhood Development

### PR DESCRIPTION
**Pull request by smarzban**

### CODE REVIEW

**Comment by smarzban** [comment]
@miltontony ready for review

File: springboard_iogt/static/css/super.css [hunk 1]
```diff
@@ -80,6 +80,7 @@
     #submenu .connectsmart a{background-image: url("img/connect_smart.png");}
     #submenu .home a{background-image: url("img/arrow_right_grey.png");}
     #submenu .allin a{background-image: url("img/hiv_all_in.png");}
+    #submenu .ecd a{background-image: url("img/early_childhood_development.png");}
```
Comment (justinvdm) [inline springboard_iogt/static/css/super.css:84]: I don't see this image in the repo, where do we get it from?

File: springboard_iogt/utils.py [hunk 2]
```diff
@@ -45,6 +45,12 @@ class ContentSection(object):
             'descriptor': _('Epidemics, disasters or conflicts'),
             'name': 'ebola'
         }),
+        ('ecd', {
+            'owner': _('Early Childhood Development'),
+            'title': _('Early Childhood Development'),
+            'descriptor': _('Care for your Child Development'),
+            'name': 'Early Life Tips'
```
Comment (justinvdm) [inline springboard_iogt/utils.py]: The ticket seems to want `Early Life Tips!` here.

File: springboard_iogt/static/css/super.css (hunk 1 above)
Comment (smarzban) [inline springboard_iogt/static/css/super.css:84]: Botswang is going to give me the image soon :)

**Comment by justinvdm** [comment]
Looks good, other than the two comments, :+1:.

Happy to wait for milton to review first though if you'd prefer, I don't know UC well enough to give a confident +1.

File: springboard_iogt/utils.py (hunk 2 above)
Comment (miltontony) [inline springboard_iogt/utils.py]: So this should be `ecd` - it's the _name of the repo_.
The `Early Life Tips` would go into the `title` field.

**Comment by smarzban** [comment]
@miltontony ready for review

**Comment by miltontony** [comment]
:+1:

**Pull request merged**